gson/src/test/java/com/baeldung/gson/jsoncompare/JsonCompareUnitTest.java

```diff
@@ -0,0 +1,93 @@
+package com.baeldung.gson.jsoncompare;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class JsonCompareUnitTest {
+    @Test
+    public void givenIdenticalSimpleObjects_whenCompared_thenEqual() {
+        JsonParser parser = new JsonParser();
+        String string1 = "{\"customer\": {\"id\": \"44521\",\"fullName\": \"Emily Jenkins\", \"age\": 27 }}";
+        String string2 = "{\"customer\": {\"id\": \"44521\", \"fullName\": \"Emily Jenkins\",\"age\": 27}}";
+
+        Assert.assertTrue(parser.parse(string1)
+            .isJsonObject());
+        Assert.assertEquals(parser.parse(string1), parser.parse(string2));
```

Comment:
Let's static import the assertions inside `org.junit.Assert` so we don't need to prefix them with `Assert.`